src/registrar/assets/js/uswds-edited.js

```diff
@@ -5949,8 +5966,10 @@ const showToolTip = (tooltipBody, tooltipTrigger, position) => {
     const topMargin = calculateMarginOffset("top", e.offsetHeight, tooltipTrigger);
     const leftMargin = calculateMarginOffset("left", e.offsetWidth, tooltipTrigger);
     setPositionClass("top");
-    e.style.left = `50%`; // center the element
-    e.style.top = `-${TRIANGLE_SIZE}px`; // consider the pseudo element
+
+    // DOTGOV: updated logic for position:fixed
```

Comment:
BLOCKING: For all changes in this file, I suggest you wrap them (even if it's one line) with DOTGOV EDIT and END DOTGOV EDIT comments. Leave the original USWDS code within these blocks, commented out.

We want to make it as easy as possible for devs who do not have context to update USWDS. To do so, they have to compile new releases and then go in and redo our edits onto the compiled code. I think my suggestion takes out some cognitive effort from this task.

Reply:
Totally agree.  I like this suggestion.